lib/Doctrine/Persistence/ObjectRepository.php

```diff
@@ -47,6 +47,7 @@ public function findAll();
      *
      * @throws UnexpectedValueException
      *
+     * @psalm-param array<string, 'asc'|'desc'|'ASC'|'DESC'> $orderBy
```

Comment:
This is fine to describe behaviour. Maybe for a future version, we could introduce constants, that way we could have Psalm guiding users to prefer uppercased values